app/models/ext_management_system.rb

```diff
@@ -142,37 +142,8 @@ def self.create_from_params(params, endpoints, authentications)
 
   serialize :options
 
-  supports_not :add_host_initiator
-  supports_not :add_storage
-  supports_not :add_volume_mapping
-  supports_not :admin_ui
-  supports_not :assume_role
-  supports_not :block_storage
-  supports_not :catalog
-  supports_not :change_password
-  supports_not :cloud_object_store_container_create
-  supports_not :cloud_subnet_create
-  supports_not :cloud_tenant_mapping
-  supports_not :cloud_volume_create
-  supports_not :storage_service_create
-  supports_not :console
-  supports_not :create_iso_datastore
-  supports_not :discovery
-  supports_not :events
-  supports_not :label_mapping
-  supports_not :metrics
-  supports_not :object_storage
-  supports_not :provisioning
-  supports_not :publish
-  supports_not :reconfigure_disks
+  # NOTE: automate (a child) does not support this
```

Comment:
automate or AutomationManager? (they are 2 different things, so want to be accurate, because it reads strange right now.  Separately, why wouldn't an automation manager support refresh ems?

Reply:
Oh I see, specifically embedded ansible (`ManageIQ::Providers::EmbeddedAnsible::AutomationManager`) doesn't support this.  I would remove this comment, because it feels like Basmati Rice - we may have others in the future and this comment will get stale, or perhaps we may change embedded ansible to allow "refresh".  For example, embedded ansible works on repositories, so maybe a "refresh" updates the content from the git repository.

Reply:
thnx.

feels like we should not give and take away.
Feels like we should only give

But I guess the embedded ansible automation manager is a strange case, so it is good.
I don't think I made another mistake like this. But there is always that possibility.

Reply:
Yeah I agree I don't think we should say we support something in a base class unless it is implemented at the base class level which this is not.

Reply:
@kbrock you want to tackle that in a follow-up since it isn't directly related to what you're doing here?